ios/xcode/OpenGLES.xcodeproj/project.pbxproj

```diff
@@ -5703,6 +6313,9 @@
 				0A936F22244CEFA800B3497E /* libangle_base.a */,
 				0A93703D244CF03700B3497E /* libangle_base_mac.a */,
 				0A937154244CF04900B3497E /* libangle_base_tvos.a */,
+				2F41094C24AA926F001EB161 /* libangle_gl_backend_tvos_13.a */,
```

Comment:
Please create `angle_metal_backend_tvos_13` target also. `angle_gl_backend_tvos_13` is only for OpenGL back-end. It's better you do it on `gles3-dev` branch since IOSurface implementation for metal is there.